### PR TITLE
Add a way to set map and path in factory

### PR DESCRIPTION
**Pull request by harikt**
This allows to pass the map array and paths to the template registry on constructor using the ViewFactory class.

### CODE REVIEW

**Comment by pmjones** [comment]
Hey @harikt -- can you add some README documentation on this? Then I can merge it and make a minor release.

**Comment by pmjones** [comment]
Or you can merge it and make the release. :-)

**Comment by harikt** [comment]
Sure, I will update the README.md . I don't like me sending a PR and merging the same for a few reasons. If the same person is reviewing the possibility of missing things are greater. So I prefer anyone other than me looking into it and verifying the same. So it is always for your consideration :+1:  . If you put a +1 I will do the next steps ;) .

**Comment by harikt** [comment]
@pmjones have added some information to README. Let me know if that helps.

Thank you

File: README.md [hunk 1]
```diff
@@ -402,3 +402,43 @@ $layout_registry = $view->getLayoutRegistry();
...
+### Advanced Configuration
+
+Alternatively you can pass [$helpers](https://github.com/harikt/Aura.View#custom-helper-managers),
+mapping information or paths for views and layouts as below.
+
+```php
+<?php
+$view_factory = new \Aura\View\ViewFactory;
+$view = $view_factory->newInstance(
+    $helpers = null,
```
Comment (jakejohns) [inline README.md:415]: Shouldn't this just be `null`, rather than an assignment?

Comment (harikt) [reply README.md:415]: You are right, it can be just null. But I wanted to show it is helpers and people can pass helpers other than null.

**Comment by harikt** [comment]
@pmjones in case you have some time. Have a look into the updated docs.

I will make another release for Aura.View once you have reviewed the same.

Thank you.

**Comment by harikt** [comment]
@pmjones ping.

**Comment by pmjones** [comment]
Done :-)

**Pull request merged**